scipy/signal/lti_conversion.py

```diff
@@ -459,6 +466,38 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
         cd = c
         dd = d
 
+    elif method == 'foh':
+        # Size parameters for convenience
+        n = a.shape[0]
+        m = b.shape[1]
+
+        # Build an exponential matrix similar to 'zoh' method
+        em_upper = linalg.block_diag(np.block([a, b]) * dt, np.eye(m))
+        em_lower = zeros((m, n + 2 * m))
+        em = np.block([[em_upper], [em_lower]])
+
+        ms = linalg.expm(em)
+
+        # Get the three blocks from upper rows
+        ms11 = ms[:n, 0:n]
+        ms12 = ms[:n, n:n + m]
+        ms13 = ms[:n, n + m:]
+
+        ad = ms11
+        bd = ms12 - ms13 + ms11 @ ms13
+        cd = c
+        dd = d + c @ ms13
+
+    elif method == 'impulse':
+        if not np.allclose(d, 0):
+            raise ValueError("Impulse method is only applicable"
+                             "to strictly proper systems")
+
+        ad = linalg.expm(a * dt)
+        bd = b * dt
```

Comment:
I think you meant `ad @ b` here and `cd = c` probably under it

Reply:
The systems `(exp(A T), B T, C exp(A T), C B T)` and `(exp(A T), exp(A T) B T, C, C B T)` are equivalent. But if you wish I can change the realization for the second one.

Reply:
They are indeed but often since these are compared with ZOH and there we have c matrices unmodified. Since we do not perform any balancing here, it is better to reflect the changes to the `b` array instead of c.